src/scripts/extensions/webExtensionBase/webExtension.ts

```diff
@@ -242,6 +243,19 @@ export class WebExtension extends ExtensionBase<WebExtensionWorker, W3CTab, numb
 		});
 	}
 
+	private registerTabCreateOrUpdateListener() {
+		WebExtension.browser.tabs.onCreated.addListener((tab: W3CTab) => {
+		if (tab.url.match("chrome://*")) {
+			WebExtension.browser.action.disable(tab.id);
+		}
+		});
+		WebExtension.browser.tabs.onUpdated.addListener((tabId: number, changeInfo: any, tab: W3CTab) => {
```

Comment:
do we need to listen to tab switch as well?

Reply:
No the listeners get created for individual tabs so a tab switch would have separate listener